bin/vm/create/index.js

```diff
@@ -109,7 +108,12 @@ module.exports = resource => Cli.createCommand('create', {
                 netadp.network = args.network;
                 netadp.service = 'private';
             } else {
-                netadp.service = 'public';
+                const services = await args.helpers.api.get('/service/');
+                const vm_service = services.find(service => service.name === args.type || service._id === args.type);
```

Comment:
In my opinion it should go to public-API.

Reply:
IMO is already there ;)